includes/minimal_header_template.php

```diff
@@ -0,0 +1,110 @@
+<?php
+// try{
+    // if($LANG_TAG == 'en' || !file_exists($SERVER_ROOT.'/content/lang/header.' . $LANG_TAG . '.php')) include_once($SERVER_ROOT . '/content/lang/header.en.php');
+    // else include_once($SERVER_ROOT . '/content/lang/header.' . $LANG_TAG . '.php');
+// } 
+// catch(Exception $error){
+//     // var_dump($error);
+// }
+include_once($SERVER_ROOT . '/includes/head.php');
+
+include_once($SERVER_ROOT . '/classes/ProfileManager.php');
+$pHandler = new ProfileManager();
+$isAccessiblePreferred = $pHandler->getAccessibilityPreference($SYMB_UID);
+$SHOULD_USE_HARVESTPARAMS = $SHOULD_USE_HARVESTPARAMS ?? false;
+$collectionSearchPage = $SHOULD_USE_HARVESTPARAMS ? '/collections/index.php' : '/collections/search/index.php';
+?>
+<div class="header-wrapper" style="position:absolute;top:10; width: 100vw;">
+	<header>
+		<div class="menu-wrapper">
+			<!-- Hamburger icon -->
+			<input class="side-menu" type="checkbox" id="side-menu" name="side-menu" />
+			<label class="hamb hamb-line hamb-label" for="side-menu" tabindex="0">☰</label>
+			<!-- Menu -->
+			<nav class="top-menu" aria-label="hamburger-nav">
```

Comment:
Hamburger menu is very fat, this is an issue with the normal header. But is also doesn't have a z-index that allows it to be viewed over the leaflet maps.

Reply:
I noticed the z-index issue on some of the map pages, but not others, and just solved it where it was a problem. E.g., that's why there's a local class override of header-wrapper in collections/georef/georefclone.php.

I added the hamburger menu size issue to Part 2 (i.e., I'm not planning on solving this currently).